mkdocs.yml

```
@@ -38,6 +38,14 @@ nav:
     - Notes: "40_development.md"
     - REST API Swagger: https://api.dandiarchive.org/swagger
     - REST API Redoc: https://api.dandiarchive.org/redoc
+    - Replicating Dandi:
+      - Getting Started: "60_initialize_vendors.md"
```

We can also have an overview/getting started page, if that would be helpful.

```suggestion
      - Initialize Vendor Accounts: "60_initialize_vendors.md"
```

---

Updated! I'm not sure what you had in mind for the "Getting Started" page -- so I just included a placeholder file and reassigned the "Initivalize Vendor Accounts"

---

Changed per @asmacdo's PR here: https://github.com/lincbrain/handbook/pull/9